container_linux_test.go

```diff
@@ -730,9 +730,7 @@ func TestShimSigkilled(t *testing.T) {
 	defer cancel()
 
 	// redis unset its PDeathSignal making it a good candidate
-	//
-	// FIXME: change this back after multiplatform support is added to pull
-	image, err = client.Pull(ctx, "docker.io/amd64/redis:alpine", WithPullUnpack)
+	image, err = client.Pull(ctx, "docker.io/library/redis:alpine", WithPullUnpack)
```

Comment:
Note that `redis:alpine` is a manifest list, but not multi-arch enabled until the underlying Alpine parent image goes multi-arch (which is being worked on right now). For tests to run cross-platform, would have to switch to `redis:latest` which is multi-arch enabled.